acceptance/bundle/templates/telemetry/custom-template/script

```diff
@@ -0,0 +1,14 @@
+trace $CLI bundle init . --config-file input.json --output-dir output
+
+# We only need to keep the output databricks.yml to assert the bundle_uuid is consistent
+# in both the generated YAML file and telemetry payload.
+mv output/my_custom_template/databricks.yml out.databricks.yml
+rm -r output
+
+
+cmd_exec_id=$(cat out.requests.txt | jq '.headers."User-Agent".[0]'| head -n 1 | grep -o 'cmd-exec-id/[^ ]*' | cut -d '/' -f2)
+bundle_uuid=$(cat out.databricks.yml | grep -o 'uuid: [^\n]*' | cut -d ' ' -f2)
+
+update_file.py out.requests.txt $cmd_exec_id  '[CMD-EXEC-ID]'
+update_file.py out.requests.txt $bundle_uuid '[BUNDLE-UUID]'
+update_file.py out.databricks.yml $bundle_uuid '[BUNDLE-UUID]'
```

Comment:
For this custom template test, I believe the intent is to make sure it only logs "custom" as the template name and nothing else. Could you include a comment explaining this intent somewhere in this directory or file?